stdlib/3/os/__init__.pyi

```diff
@@ -401,17 +567,10 @@ def killpg(pgid: int, sig: int) -> None: ...  # Unix only
 def nice(increment: int) -> int: ...  # Unix only
 def plock(op: int) -> None: ...  # Unix only ???op is int?
 
-if sys.version_info >= (3, 0):
-    class popen(_TextIOWrapper):
-        # TODO 'b' modes or bytes command not accepted?
-        def __init__(self, command: str, mode: str = ...,
-                     bufsize: int = ...) -> None: ...
-        def close(self) -> Any: ...  # may return int
-else:
```

Comment:
Let's leave this in so I can finish my project to merge the Python 2 and 3 `os` stubs.